### DISCUSSION TITLE
designable使用switch组件出现Warning

### DISCUSSION THREAD

**Discussion by aloha66** [alibaba/formily#1890] [Q&A 2.x]
[复现链接](https://designable-antd.formilyjs.org/)
![image](https://user-images.githubusercontent.com/20942781/127270051-5edc93d7-1f7d-41fc-937d-a33b273a6f76.png)
src\schemas\Switch.ts的optionType: 'button',还没有去除

另外，switch的size只有两个可选值default，small，能不能去掉大这个选项？

还有如果提issus，貌似在选择包的时候没发现有designable选项

**Comment by janryWang** [comment]
去不掉大选项，size的国际化描述是统一的

**Comment by janryWang** [comment]
option button的你提个PR改掉呗

**Reply by aloha66** [reply]
好吧 大佬，我提pr了
